src/ConnectionFromMongoCursor.js

```diff
@@ -78,6 +78,12 @@ export const calculateOffsets = ({ args, totalCount }: OffsetOptions) => {
 
   const limit = endOffset - startOffset;
 
+  mongodbCursor.skip(skip);
+  mongodbCursor.limit(limit);
+
+  const endCursorOffset = limit + skip;
+  const startCursorOffset = skip;
```

Comment:
where are these been used?

Reply:
When we need to move from one page to another. 
I think... 

Example:

UserList 

`firstItemIndex` and `lastItemIndex`.